### ISSUE TITLE
Possible sfn lint bug

### ISSUE THREAD

**Issue by webframp** [sparkleformation/sfn#254]
Not sure if this is an error in usage or an actual bug somewhere in the sparkle_formation lib, but we only see this with the `sfn lint` command

Trying out some nesting examples from here:

http://www.sparkleformation.io/docs/sparkle_formation/compile-time-parameters.html#template-nesting

Like this: 
```rb
SparkleFormation.new(:test,
  :compile_time_parameters => {
    :number_of_nodes => {
      :type => :number,
      :default => 1
    }
  }
) do

  state!(:number_of_nodes).times do |i|
    dynamic!(:ec2_instance, "node_#{i}")
  end

end
```

If you create the example above, directly as the SFN docs say, it fails.
```
bundle exec sfn lint --file generic_ec2instances
[Sfn]: Callback after_config lambda: starting
[Sfn]: Callback after_config lambda: complete
[Sfn]: NumberOfNodes [1]:
ERROR: NoMethodError: undefined method `validate_parameter' for #<Sfn::Command::Lint:0x5071e90>
Did you mean?  validate_arguments!
```

**Comment by CoreyLivingston** [comment]
This is where it is hanging:
```Ruby
validation = validate_parameter(answer, param_value)
```
https://github.com/sparkleformation/sfn/blob/3f24ab023386bdc6dba7189749cf66dab814b873/lib/sfn/command_module/template.rb#L85

**Comment by webframp** [comment]
This is fixed in #248 it looks like and I forgot I had previously reported in #247, closing this one
